### PR TITLE
Booking api tests with python support

### PR DESCRIPTION
**Pull request by therearesomewhocallmetim**
Arsenty's pull request with Booking api test getting data from the local test server instead of the booking.com server, and the modifications to the testserver to support that.

### CODE REVIEW

**Comment by milchakov** [comment]
DUDE JTTP

File: tools/python/jsons.py [hunk 1]
```diff
@@ -183,3 +183,28 @@
...
+            },
+            {
+                "currency_code":"ISK",
+                "max_price":"500000.00",
+                "min_price":"30000.00"
+            }
+        ]
+    }
+]
+"""
```
Comment (milchakov) [inline tools/python/jsons.py]: пустую строчку в конец файла

**Comment by therearesomewhocallmetim** [comment]
DUDE JTTP

File: tools/python/jsons.py (hunk 1 above)
Comment (therearesomewhocallmetim) [inline tools/python/jsons.py]: DONE

**Comment by therearesomewhocallmetim** [comment]
DUDE JTTP

**Comment by therearesomewhocallmetim** [comment]
LGTM

**Pull request merged**